apps/cli/src/commands/boot-challenger.ts

```diff
@@ -45,11 +45,13 @@ let cachedSigner: {
 let cachedWebhookUrl: string | undefined;
 let cachedSecretKey: string;
 let lastAssertionTime: number;
+let lastSubmittedAssertionId: number = 0;
```

Comment:
Why do we need this in memory ?
This does not seem to be initialized ever, I don't think we want to make sure this is synced in all events but rather get it from onChain ?

Reply:
This has been removed and a core function added to retrieve the last assertionId and timestamp from chain.